adoc/admin-security-rbac.adoc

```diff
@@ -98,7 +98,11 @@ It is a known limitation of the addon customization process.
 Administrators can update the authentication connector settings after {productname}
 deployment as follows:
 
+<<<<<<< Updated upstream
```

Comment:
Somehow I don't think those are meant to be here...

Reply:
Nope, thanks for pointing that out.This is a draft, I need to have a proper look at this. :)